packages/astro/test/fixtures/core-image-svg/astro.config.mjs

```diff
@@ -4,6 +4,6 @@ import { defineConfig } from 'astro/config';
 export default defineConfig({
   integrations: [mdx()],
 	experimental: {
-		svg: {}
```

Comment:
This way of passing the option shouldn't be valid

Reply:
IIUC passing `{}` is like passing `true`. Couldn't we support `{}` without specifying `mode` too? It feels to me futureproofing if it ever has more optional options in the object, so you don't have to specify `mode` always.

Reply:
I applied the change in [`26c65f2` (#12625)](https://github.com/withastro/astro/pull/12625/commits/26c65f25c7700ad356b0c358296873f0080c2cec) and tested that `svg: {}` is still a valid configuration

Reply:
It doesn't mean a lot currently, but it's futureproofing if we add more options to it in the future, so I think it's an ok tradeoff for now as long as we don't recommend people to enable with `{}` only.